src/debug.c

```diff
@@ -1241,6 +1252,10 @@ static void *getAndSetMcontextEip(ucontext_t *uc, void *eip) {
 
 VALKEY_NO_SANITIZE("address")
 void logStackContent(void **sp) {
+    if (server.hide_user_data_from_log) {
```

Comment:
Sorry I am a little bit confused, in valkey.conf, it is written:  disable this feature by changing the config value to -1. But here if we set server.hide_user_data_from_log as -1,  the condition is still true.
If you mean disable this feature by changing the config value to 0 ?